### PR TITLE
Changes for transition from DevOpsACCELERATE to OutagePREVENT platform.

### PR DESCRIPTION
**Pull request by angelmoo**
Changes for:
adding transition from DevOpsACCELERATE api to OutagePREVENT

updated test cases.

all test cases ran well.

### CODE REVIEW

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnector.java [hunk 1]
```diff
@@ -107,66 +109,33 @@ public void authenticate(Authentication auth) throws VCertException {
...
+	  
+	  CertificateIssuingTemplate cit = null;
+	    String zoneId = null;
+	    if((appName != null && appName != "") && (citAlias != null && citAlias != "")) {
+	    	
+	    	 cit = cloud.certificateIssuingTemplateByAppNameAndCitAlias(appName, citAlias, auth.apiKey());
+	    	 
+	    	
+	    }else {
+	    	  throw new VCertException("The parameters: appName, citAlia or both are empty");
```
Comment (tr1ck3r) [inline src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnector.java:125]: Typo: citAlia -> citAlias

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/Cloud.java [hunk 2]
```diff
@@ -3,84 +3,71 @@
...
+  @RequestLine("POST  /outagedetection/v1/certificaterequests")
   CloudConnector.CertificateRequestsResponse certificateRequest(@Param("apiKey") String apiKey,
       CloudConnector.CertificateRequestsPayload csr);
 
   @Headers("tppl-api-key: {apiKey}")
-  @RequestLine("GET /certificaterequests/{id}")
+  @RequestLine("GET  /outagedetection/v1/certificaterequests/{id}")
   CertificateStatus certificateStatus(@Param("id") String id, @Param("apiKey") String apiKey);
 
   @Headers("tppl-api-key: {apiKey}")
```
Comment (tr1ck3r) [inline src/main/java/com/venafi/vcert/sdk/connectors/cloud/Cloud.java:59]: Are we missing the "Content-Type: text/plain" header here?

**Review by tr1ck3r — COMMENTED** [review]
Looks good @angelmoo, just a couple of small comments.  I want to test the latest build jar before approving.

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnector.java (hunk 1 above)
Comment (angelmoo) [reply src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnector.java]: fixed on the latest commit.

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/Cloud.java (hunk 2 above)
Comment (angelmoo) [reply src/main/java/com/venafi/vcert/sdk/connectors/cloud/Cloud.java:59]: Actually I forgot to remove that headers on:   Response certificateAsPem(@Param("id") String id, @Param("apiKey") String apiKey), now removed it on that so now we are consistent

**Review by tr1ck3r — APPROVED** [review]

File: src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java [hunk 3]
```diff
@@ -115,5 +118,25 @@ public static void addCustomFieldsToRequest( CertificateRequest request, Certifi
...
 		}
 	}
+	
+	public static String getIpAddress() throws UnknownHostException {
+		
+		return  Inet4Address.getLocalHost().getHostAddress();
+		
+	}
+
+	public static void addApliClientInformation( com.venafi.vcert.sdk.connectors.cloud.CloudConnector.CertificateRequestsPayload payload ) {
```
Comment (rvelaVenafi) [inline src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java:128]: minor typo here. Method name should be addApiClientInformation()

Comment (angelmoo) [reply src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java]: fixed on the latest commit.

**Review by rvelaVenafi — APPROVED** [review]

**Pull request merged**